docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -160,16 +162,21 @@ yesnomaybe: have_docket_copies_already
 ---
 id: docket_warning
 field: wants_dockets_first
-question: ''
+question: Why Get Docket Sheets?
 subquestion: |
   There are times when it is important to have certified copies of
   your docket sheets for all your cases.
 
+  If you seal your records you won't be able to get
+  certified copies of your docket sheets until you
+  unseal them again. Unsealing your records can
+  take a long time.
```

Comment:
Note to self (todo): 'long time' might be too extreme for describing the current process of unsealing.

Reply:
I made an issue for this too. #52